spec/features/idv/doc_auth/send_link_step_spec.rb

```diff
@@ -18,6 +18,10 @@
   let(:fake_analytics) { FakeAnalytics.new }
   let(:fake_attempts_tracker) { IrsAttemptsApiTrackingHelper::FakeAttemptsTracker.new }
 
+  it "defaults phone to user's 2fa phone numebr" do
+    expect(page.find('input[name="doc_auth[phone]"]').value).to eql('+1 202-555-1212')
```

Comment:
Minor: In feature specs, I think it'd end up being more maintainable and more "true" to the user's experience (and helping assure accessible linking) to look for the field by its visible label.

i.e.

```suggestion
    expect(page.find_field(t('two_factor_authentication.phone_label')).value).to eq('+1 202-555-1212')
```